wandb/sdk/artifacts/artifact.py

```diff
@@ -1386,18 +1400,30 @@ def do_write(f: IO) -> None:
         return entry
 
     def _add_local_file(
-        self, name: StrPath, path: StrPath, digest: Optional[B64MD5] = None
+        self,
+        name: StrPath,
+        path: StrPath,
+        digest: Optional[B64MD5] = None,
+        policy: Optional[Literal["mutable", "immutable"]] = "mutable",
```

Comment:
based on your check, doesn't look like policy is Optional

Reply:
fixed it, thanks for catching this